db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -0,0 +1,188 @@
+#
+# Description: <Method description here>
+#
+def request_info
+  @service = ($evm.root['vmdb_object_type'] == 'service_template_provision_task') ? true : false
+  @miq_request = $evm.root['miq_request']
+  $evm.log(:info, "Request: #{@miq_request.description} id: #{@miq_request.id} ")
+end
+
+def calculate_requested(options_hash = {})
+  {:memory  => get_total_requested(options_hash, :vm_memory),
+   :cpu     => get_total_requested(options_hash, :cores_per_socket),
+   :storage => get_total_requested(options_hash, :storage),
+   :vms     => get_total_requested(options_hash, :number_of_vms)}
+end
+
+def get_total_requested(options_hash, prov_option)
+  total_requested = collect_template_totals(prov_option)
+  total_requested = request_totals(total_requested.to_i,
+                                   collect_dialog_totals(prov_option, options_hash).to_i) if options_hash
+  total_requested
+end
+
+def request_totals(template_totals, dialog_totals)
+  template_totals < dialog_totals ? dialog_totals : template_totals
+end
+
+def collect_template_totals(prov_option)
+  total = collect_totals(service_prov_option_value(prov_option)) if @service
```

Comment:
@tinaafitz 
can we use
@service ? collect_totals(service_prov_option_value(pro_option)) : collect_totals(vm_prov_option_value(prov_option))